src/subgraphs.ts

```diff
@@ -67,3 +67,15 @@ export const BOOSTED_MASTERCHEF_SUBGRAPH: {[chainId in ChainId]: string } = {
     [ChainId.FUJI]: ZERO_ADDRESS,
     [ChainId.AVALANCHE]: 'https://api.thegraph.com/subgraphs/name/traderjoe-xyz/boosted-master-chef'
 }
+
+export const NFT_CONTRACTS_SUBGRAPH: {[chainId in ChainId]: string } = {
+    [ChainId.RINKEBY]: 'https://api.thegraph.com/subgraphs/name/traderjoe-xyz/nft-contracts-rinkeby', 
+    [ChainId.FUJI]: ZERO_ADDRESS,
```

Comment:
we have a Fuji one actually: https://api.thegraph.com/subgraphs/name/traderjoe-xyz/nft-contract-fuji